<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>broker_mainwindow</class>
 <widget class="QWidget" name="broker_mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListView" name="brokerMainView">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>531</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="requestsButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>490</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Заказы</string>
   </property>
  </widget>
  <widget class="QPushButton" name="companyButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>490</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Компании</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addCompanyButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteCompanyButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>100</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changeCompanyButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>100</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
  <widget class="QListWidget" name="companyWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>531</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="handleWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>531</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="handleRequestButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Обработать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelRequestButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>100</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Отклонить</string>
   </property>
  </widget>
  <widget class="QListWidget" name="serviceWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>531</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteServiceButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>100</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addServiceButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changeServiceButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>100</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
  <widget class="QListView" name="companyAddView">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>130</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addCompanyButtonLast">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>190</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelAddCompanyButton">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>190</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Отменить</string>
   </property>
  </widget>
  <widget class="QLabel" name="companyNameLabel">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>150</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Компания</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="companyNameLineEdit">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>150</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="changeCompanyButtonLast">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>190</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
